### ISSUE TITLE
Program lacks a build-in help

### ISSUE THREAD

**Issue by the-Arioch** [Genoil/ZECMiner#36]
The homepage claims: 

> Command line options are very limited:
> help, h: show this help message

This claim, however, seems not being held.

```
b:\zc\flypool>genoil.exe
Genoil's ZEC miner 0.4 - ft. Silent Army
========================================
using Silent Army kernels by mrb

Made possible by tromp, ocminer (suprnova), feeleep (coinmine.pl) and 110GH

Thanks for your donation! (BTC)
Silent Army: 1DbGfE695VUs3kcsy6cX6sLjTmsPawmwvj
Genoil : 1Nu2fMCEBjmnLzqb8qUJpKgq5RoEWFhNcW

error: the option '--conn' is required but missing

b:\zc\flypool>genoil.exe --help
Genoil's ZEC miner 0.4 - ft. Silent Army
========================================
using Silent Army kernels by mrb

Made possible by tromp, ocminer (suprnova), feeleep (coinmine.pl) and 110GH

Thanks for your donation! (BTC)
Silent Army: 1DbGfE695VUs3kcsy6cX6sLjTmsPawmwvj
Genoil : 1Nu2fMCEBjmnLzqb8qUJpKgq5RoEWFhNcW

error: the option '--conn' is required but missing

b:\zc\flypool>genoil.exe -h
Genoil's ZEC miner 0.4 - ft. Silent Army
========================================
using Silent Army kernels by mrb

Made possible by tromp, ocminer (suprnova), feeleep (coinmine.pl) and 110GH

Thanks for your donation! (BTC)
Silent Army: 1DbGfE695VUs3kcsy6cX6sLjTmsPawmwvj
Genoil : 1Nu2fMCEBjmnLzqb8qUJpKgq5RoEWFhNcW

error: the option '--conn' is required but missing

b:\zc\flypool>genoil.exe -help
Genoil's ZEC miner 0.4 - ft. Silent Army
========================================
using Silent Army kernels by mrb

Made possible by tromp, ocminer (suprnova), feeleep (coinmine.pl) and 110GH

Thanks for your donation! (BTC)
Silent Army: 1DbGfE695VUs3kcsy6cX6sLjTmsPawmwvj
Genoil : 1Nu2fMCEBjmnLzqb8qUJpKgq5RoEWFhNcW

error: unrecognised option '-help'
```

**Comment by the-Arioch** [comment]
Also

```
b:\zc\flypool>genoil.exe -?
Genoil's ZEC miner 0.4 - ft. Silent Army
========================================
using Silent Army kernels by mrb

Made possible by tromp, ocminer (suprnova), feeleep (coinmine.pl) and 110GH

Thanks for your donation! (BTC)
Silent Army: 1DbGfE695VUs3kcsy6cX6sLjTmsPawmwvj
Genoil : 1Nu2fMCEBjmnLzqb8qUJpKgq5RoEWFhNcW

error: unrecognised option '-?'

b:\zc\flypool>genoil.exe h
Genoil's ZEC miner 0.4 - ft. Silent Army
========================================
using Silent Army kernels by mrb

Made possible by tromp, ocminer (suprnova), feeleep (coinmine.pl) and 110GH

Thanks for your donation! (BTC)
Silent Army: 1DbGfE695VUs3kcsy6cX6sLjTmsPawmwvj
Genoil : 1Nu2fMCEBjmnLzqb8qUJpKgq5RoEWFhNcW

error: the option '--conn' is required but missing

b:\zc\flypool>genoil.exe help
Genoil's ZEC miner 0.4 - ft. Silent Army
========================================
using Silent Army kernels by mrb

Made possible by tromp, ocminer (suprnova), feeleep (coinmine.pl) and 110GH

Thanks for your donation! (BTC)
Silent Army: 1DbGfE695VUs3kcsy6cX6sLjTmsPawmwvj
Genoil : 1Nu2fMCEBjmnLzqb8qUJpKgq5RoEWFhNcW

error: the option '--conn' is required but missing

b:\zc\flypool>genoil.exe /help
Genoil's ZEC miner 0.4 - ft. Silent Army
========================================
using Silent Army kernels by mrb

Made possible by tromp, ocminer (suprnova), feeleep (coinmine.pl) and 110GH

Thanks for your donation! (BTC)
Silent Army: 1DbGfE695VUs3kcsy6cX6sLjTmsPawmwvj
Genoil : 1Nu2fMCEBjmnLzqb8qUJpKgq5RoEWFhNcW

error: the option '--conn' is required but missing

b:\zc\flypool>genoil.exe /h
Genoil's ZEC miner 0.4 - ft. Silent Army
========================================
using Silent Army kernels by mrb

Made possible by tromp, ocminer (suprnova), feeleep (coinmine.pl) and 110GH

Thanks for your donation! (BTC)
Silent Army: 1DbGfE695VUs3kcsy6cX6sLjTmsPawmwvj
Genoil : 1Nu2fMCEBjmnLzqb8qUJpKgq5RoEWFhNcW

error: the option '--conn' is required but missing

b:\zc\flypool>genoil.exe /?
Genoil's ZEC miner 0.4 - ft. Silent Army
========================================
using Silent Army kernels by mrb

Made possible by tromp, ocminer (suprnova), feeleep (coinmine.pl) and 110GH

Thanks for your donation! (BTC)
Silent Army: 1DbGfE695VUs3kcsy6cX6sLjTmsPawmwvj
Genoil : 1Nu2fMCEBjmnLzqb8qUJpKgq5RoEWFhNcW

error: the option '--conn' is required but missing

b:\zc\flypool>genoil.exe -?
Genoil's ZEC miner 0.4 - ft. Silent Army
========================================
using Silent Army kernels by mrb

Made possible by tromp, ocminer (suprnova), feeleep (coinmine.pl) and 110GH

Thanks for your donation! (BTC)
Silent Army: 1DbGfE695VUs3kcsy6cX6sLjTmsPawmwvj
Genoil : 1Nu2fMCEBjmnLzqb8qUJpKgq5RoEWFhNcW

error: unrecognised option '-?'

b:\zc\flypool>genoil.exe --?
Genoil's ZEC miner 0.4 - ft. Silent Army
========================================
using Silent Army kernels by mrb

Made possible by tromp, ocminer (suprnova), feeleep (coinmine.pl) and 110GH

Thanks for your donation! (BTC)
Silent Army: 1DbGfE695VUs3kcsy6cX6sLjTmsPawmwvj
Genoil : 1Nu2fMCEBjmnLzqb8qUJpKgq5RoEWFhNcW

error: unrecognised option '--?'

b:\zc\flypool>genoil.exe ?
Genoil's ZEC miner 0.4 - ft. Silent Army
========================================
using Silent Army kernels by mrb

Made possible by tromp, ocminer (suprnova), feeleep (coinmine.pl) and 110GH

Thanks for your donation! (BTC)
Silent Army: 1DbGfE695VUs3kcsy6cX6sLjTmsPawmwvj
Genoil : 1Nu2fMCEBjmnLzqb8qUJpKgq5RoEWFhNcW

error: the option '--conn' is required but missing
```

**Comment by zenjakes** [comment]
looks like you forgot the -c parameter?

**Comment by Genoil** [comment]
Fixed 0.5
